Scripts/GameOverController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+
+/// <summary>
+/// After player is died then on hitting of restart button it will load level-1.
+/// </summary>
+namespace Outscal.BasicUnity2DProject
+{
+    public class GameOverController : MonoBehaviour
+    {
+        [Header("Button Settings")]
+        [SerializeField] private Button buttonRestart;
+
+        private void Awake()
+        {
+            buttonRestart.onClick.AddListener(ReloadLevel);
+        }
+
+        public void PlayerDied()
+        {
+            SoundManager.Instance.PlayMusic(Sounds.PlayerDeath);
+            gameObject.SetActive(true);
+        }
+
+        public void ReloadLevel()
+        {
+            SceneManager.LoadScene("Level1");

```

Comment:
Do not use constant name here use it in the starting of the class or use it from the other class.